gravitee-apim-console-webui/src/components/alerts/alert/alert.component.ts

```diff
@@ -171,6 +173,20 @@ const AlertComponent: ng.IComponentOptions = {
         $state.go('management.settings.alerts.list');
       }
     };
+
+    this.hasPermissionForCurrentScope = (permission: string): boolean => {
```

Comment:
Suggestion : would it be relevant to put this function in user.service, instead of duplicating this code ? (Or another relevant service ?)

Reply:
to be honest, I wanted to refactor it to have it in a alert helper (for example)
The problem with that is we need the $state to get the apiId or appId param and be able to know which scope to choose.
as discussed with @gaetanmaisse, we prefer to keep the $state info inside this component, instead of injecting it in a new class. I don't know if passing it as a param would resolve the thing
wdyt ?

Reply:
it's ok to me